packages/blocks/src/api/utils.js

```diff
@@ -161,19 +161,25 @@ export function getBlockLabel( blockType, attributes, context = 'visual' ) {
  * @param {Object}  attributes             The values of the block's attributes.
  * @param {?number} position               The position of the block in the block list.
  * @param {string}  [direction='vertical'] The direction of the block layout.
+ * @param {number} total The total number of blocks.
+ * @param {string} parentTitle In the event of a child block, this param gets the title of the parent block.
  *
  * @return {string} The block label.
  */
 export function getAccessibleBlockLabel(
 	blockType,
 	attributes,
 	position,
-	direction = 'vertical'
+	direction = 'vertical',
+	total,
+	parentTitle
 ) {
 	// `title` is already localized, `label` is a user-supplied value.
 	const { title } = blockType;
 	const label = getBlockLabel( blockType, attributes, 'accessibility' );
 	const hasPosition = position !== undefined;
+	const parentTitleOutput =
+		parentTitle !== undefined ? ` Child of ${ parentTitle } block.` : '';
```

Comment:
It seems like the string should be merged with the string  in line 194 to avoid concatenation issues (concatenation is not really allowed in translated strings, we prefer sprintf). which also mean we might need two strings depending on whether there's a parent.

Reply:
Actually  it seems we need the same logic for all the places that use `parentTitleOutput`

Reply:
Implemented this, it will now use conditionals rather than outputting parentTitleOutput. It's not near as clean in my opinion, but whatever helps me follow coding standards, I'll just have to get used to it.

Reply:
Thanks :) it's actually not about  coding standards but more about i18n. The translators wouldn't be able to translate the  strings otherwise.

Reply:
Ah I see. That actually makes sense.